corva/models/stream.py

```diff
@@ -155,6 +213,8 @@ def store_cache_data(self, cache_data: StreamStateData) -> int:
     @pydantic.root_validator(pre=True)
     def check_one_active_filter_at_most(cls, values):
         if values['filter_by_timestamp'] and values['filter_by_depth']:
-            raise ValueError('filter_by_timestamp and filter_by_depth can\'t be set to True together.')
+            raise ValueError(
```

Comment:
Why don't we make this `filter_mode` and it can be either `timestamp` or `depth`?

Reply:
good idea, added `filter_mode`